charts/base-cluster/templates/monitoring/tracing/grafana-tempo.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if and .Values.monitoring.tracing.enabled .Values.monitoring.prometheus.enabled -}}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: grafana-tempo
+  namespace: monitoring
+  labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: tracing
+    app.kubernetes.io/part-of: monitoring
+spec:
+  chart:
+    spec: {{- include "base-cluster.helm.chartSpec" (dict "repo" "bitnami" "chart" "grafana-tempo" "context" $) | nindent 6 }}
+  interval: 1h
+  {{- if .Values.monitoring.prometheus.enabled }}
```

Comment:
Isn't this check redundant?

In line one the whole Helmrelease is only created when prometheus is enabled, and here you check if prometheus is enabled again to set the dependsOn. But if this statement ever gets evaluatet it's always true, or am i wrong?